docs/upgrade/xenial_backup_install_restore.rst

```diff
@@ -0,0 +1,301 @@
+
+Before beginning the restore procedure, you should stop and start the Tails
+network connection using the panel widget in the upper-right corner of the
+screen. This will force the Tails Tor proxy to load the config changes made by
+the ``./securedrop-admin tailsconfig`` command. Once Tor has reconnected, you're
+ready to restore the backup.
+
+To restore from backup, run the following commands in a terminal, substituting
+the name of the backup file that you created earlier for
+`sd-backup-<date>.tar.gz`:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ ./securedrop-admin restore sd-backup-<date>.tar.gz
+
+Once the restore process is complete, the previous instance's *Application
+Server* ATHS and THS files should be copied into place on the *Admin
+Workstation*. From a terminal, run the following commands:
+
+.. code:: sh
+
+ cd ~/Persistent
+ cp app_service/app*ths securedrop/install_files/ansible-base/
+
+Finally, run the ``tailsconfig`` command again to update the *Admin
+Workstation*'s SSH configuration and desktop shortcuts:
+
+.. code:: sh
+
+ cd ~/Persistent/securedrop
+ ./securedrop-admin tailsconfig
+
+Once the command completes, stop and restart the network connection again to
+force the Tails Tor proxy to pick up on the configuration changes.
+
+Step 6: Cut over to the new instance
+------------------------------------
+
+If you used your existing instance's hardware for the Xenial installation, your
+new instance is now available, with unchanged Onion URLs. If you installed onto
+new hardware, you should now power down your old instance hardware and reboot
+your new instance's servers. Once the reboot is complete, move on to Step 7.
+
+Step 7: Test the instance connectivity
+--------------------------------------
+
+Your Xenial-based instance should now be up and running, with the *Journalist*
+and *Source* interfaces available under their original Onion URLs. To confirm
+this, use the desktop shortcuts on the *Admin Workstation* to connect to each
+interface, and log into the *Journalist Interface* using your existing
+credentials. You should also verify SSH connectivity to the *Application* and
+*Monitor* servers from a terminal, using the commands ``ssh app`` and ``ssh
+mon`` respectively.
+
+.. _bir_contact_us:
+
+Contact us
+----------
+
+If you have questions or comments regarding this process, or if you
+encounter any issues, you can always contact us by the following means:
+
+- via our `Support Portal <https://support.freedom.press>`_, if you are a member
+  (membership is approved on a case-by-case basis);
+- via securedrop@freedom.press
+  (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__)
+  for sensitive security issues (please use judiciously);
+- via our `community forums <https://forum.securedrop.org>`_.
+
+If you encounter problems that are not security-sensitive, we also encourage you
+to `file an issue <https://github.com/freedomofpress/securedrop/issues/new/>`
```

Comment:
this link needs a `_` after the final backtick (it's not rendered as a link currently)